cmd/oras/push.go

```diff
@@ -31,10 +31,12 @@ type pushOptions struct {
 	pathValidationDisabled bool
 	verbose                bool
 
-	debug    bool
-	configs  []string
-	username string
-	password string
+	debug     bool
+	configs   []string
+	username  string
+	password  string
+	insecure  bool
+	PlainHTTP bool
```

Comment:
Can we follow the pattern and keep this private? `plainHTTP`?

Reply:
Fix it.